cnd/src/ethereum.rs

```diff
@@ -86,7 +204,70 @@ impl<T: Into<Vec<u8>>> From<T> for Bytes {
 #[cfg(test)]
 mod tests {
     use super::Log;
-    use crate::ethereum::TransactionReceipt;
+    use crate::ethereum::{Address, Hash, TransactionReceipt};
```

Comment:
Nit: If you do `use super::*` in `tests` modules, you automatically get all types from the parent module which are very likely the ones you want to test :)